src/templates/globals/meta/index.tsx

```diff
@@ -0,0 +1,134 @@
+    />
+    <meta
+      name="smartbanner:icon-google"
+      content="https://play-lh.googleusercontent.com/pxpOEzV8odLxElJtNrPhorM48TPh6p-RnTWqwvlhetuMM3lZuGmbXP2_4YHlQ-fCOA=w240-h480-rw"
+    />
+    <meta name="smartbanner:button" content="VIEW" />
+    <meta
+      name="smartbanner:button-url-apple"
+      content="https://apps.apple.com/us/app/va-health-and-benefits/id1559609596"
+    />
+    <meta
+      name="smartbanner:button-url-google"
+      content="https://play.google.com/store/apps/details?id=gov.va.mobileapp&hl=en_US&gl=US&pli=1"
+    />
+    <meta name="smartbanner:enabled-platforms" content="android,ios" />
+    <meta name="smartbanner:close-label" content="Close" />
+  </Head>
+)
+
+// TODO: Implement article tags
+// TODO: Is this necessary? Are these ever set?
+// See: https://github.com/department-of-veterans-affairs/content-build/blob/f898e20d02cbf011e6e26976de95c5d33eace1c0/src/site/includes/metatags.drupal.liquid#L58-L63C12
+const ArticleTags = ({ tags }: { tags: string[] }) => <></>
+
+const CustomTags = ({ tags }: { tags: MetaTag[] }) => (
+  <Head>
+    {tags.map?.(({ tag: Tag, attributes }, i) =>
+      attributes.name === 'title' ? (
+        <title key={i}>{attributes.content}</title>
+      ) : (
+        <Tag key={i} {...attributes} />
+      )
+    )}
+  </Head>
+)
+
+// TODO: Implement default meta tags when no custom tags are present
+// TODO: Is this necessary? Are there content types where custom meta tags are not set?
+// See: https://github.com/department-of-veterans-affairs/content-build/blob/f898e20d02cbf011e6e26976de95c5d33eace1c0/src/site/includes/metatags.drupal.liquid#L88-L146
+const DefaultTags = ({
+  resource,
+}: {
+  resource: StaticPropsResource<FormattedResource>
+}) => <></>
+
+export const Meta = ({
+  resource,
+}: {
+  resource: StaticPropsResource<FormattedResource>
+}) => {
+  const noIndex = process.env.NEXT_PUBLIC_BUILD_TYPE !== environments.PROD
+  const canonicalLink =
+    'canonicalLink' in resource ? resource.canonicalLink : resource.entityPath
+
+  // TODO: pass this value when we have it present in future content types
+  const lastUpdated = null
+
+  // TODO: calculate when to show iOS banner
+  // See: https://github.com/department-of-veterans-affairs/content-build/blob/f898e20d02cbf011e6e26976de95c5d33eace1c0/src/site/filters/liquid.js#L1741-L1756
+  const showIOSBanner = false
+
+  return (
+    <>
+      <Head>
+        {noIndex && <meta name="robots" content="noindex" />}
+        <meta charSet="utf-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="HandheldFriendly" content="True" />
+        <meta name="MobileOptimized" content="320" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link
+          rel="canonical"
+          href={generateAbsoluteUrlFromEnv(canonicalLink)}
```

Comment:
I'm not sure that `generateAbsoluteUrlFromEnv()` is working as intended

![Screenshot 2023-12-04 at 8 29 55 PM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/35218fde-2176-4df5-917a-aea044c77b9c)
Both `og:url` and the `<link rel="canonical" href="...">` seem to be relative only, which returns if the build type is not valid (currently `NEXT_PUBLIC_BUILD_TYPE`)

Reply:
This relates to [the discussion in Slack](https://dsva.slack.com/archives/C01SR56755H/p1701743739547139?thread_ts=1701742110.519999&cid=C01SR56755H). 

This will not work right now. We need to do some work to nail down what different ENV vars mean, and how we are going to use them. 

This was an initial pass, but I'm now thinking we should not be using `NEXT_PUBLIC_BUILD_TYPE` or `NEXT_PHASE` to generate absolute URLs. A dev vs. prod build says very little about _where_ the build will ultimately live. We can do a production build locally, and we should still be able to generate local URLs.

Reply:
yes, completely agree

I think ultimately we will need some separate options handling from the .env files, something like: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15322 (analogous to the cli options available to content build https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/stages/build/options.js )